### ISSUE TITLE
[TypeError] Symfony\Component\Process\Process::__construct(): Argument #1 ($command) must be of type array, string given

### ISSUE THREAD

**Issue by bytehead** [contao-community-alliance/composer-plugin#87]
Happens during install with `composer` in version `2.3.0` (tested on PHP 8.1):

```
  ....
  - Installing menatwork/contao-multicolumnwizard-bundle (3.5.2): Extracting archive
  - Installing terminal42/contao-changelanguage (3.2.9): Extracting archive
  - Installing terminal42/contao-easy_themes (2.2.11): Extracting archive
  - Installing terminal42/dc_multilingual (4.2.2): Extracting archive
  - Installing toflar/psr6-symfony-http-cache-store (3.0.1): Extracting archive
    Install of codefog/contao-haste failed
    Install of terminal42/contao-easy_themes failed
    Install of terminal42/dc_multilingual failed
In Process.php line 143:
                                                                               
  [TypeError]                                                                  
  Symfony\Component\Process\Process::__construct(): Argument #1 ($command) mu  
  st be of type array, string given, called in /vendor/con  
  tao-community-alliance/composer-plugin/src/UserFilesLocator.php on line 99   
                                                                               
Exception trace:
  at phar:///composer.phar/vendor/symfony/process/Process.php:143
 Symfony\Component\Process\Process->__construct() at /vendor/contao-community-alliance/composer-plugin/src/UserFilesLocator.php:99
 ContaoCommunityAlliance\Composer\Plugin\UserFilesLocator->getPathFromConsole() at /vendor/contao-community-alliance/composer-plugin/src/UserFilesLocator.php:60
 ContaoCommunityAlliance\Composer\Plugin\UserFilesLocator->locate() at /vendor/contao-community-alliance/composer-plugin/src/Installer/AbstractModuleInstaller.php:234
 ContaoCommunityAlliance\Composer\Plugin\Installer\AbstractModuleInstaller->getFilesRoot() at /vendor/contao-community-alliance/composer-plugin/src/Installer/AbstractModuleInstaller.php:117
 ContaoCommunityAlliance\Composer\Plugin\Installer\AbstractModuleInstaller->ContaoCommunityAlliance\Composer\Plugin\Installer\{closure}() at phar:///composer.phar/vendor/react/promise/src/FulfilledPromise.php:28
 React\Promise\FulfilledPromise->then() at phar:///composer.phar/vendor/react/promise/src/Promise.php:134
 React\Promise\Promise::React\Promise\{closure}() at phar:///composer.phar/vendor/react/promise/src/Promise.php:168
 React\Promise\Promise->settle() at phar:///composer.phar/vendor/react/promise/src/Promise.php:231
 React\Promise\Promise::React\Promise\{closure}() at phar:///composer.phar/vendor/react/promise/src/FulfilledPromise.php:42
 React\Promise\FulfilledPromise->done() at phar:///composer.phar/vendor/react/promise/src/Promise.php:135
 React\Promise\Promise::React\Promise\{closure}() at phar:///composer.phar/vendor/react/promise/src/Promise.php:168
 React\Promise\Promise->settle() at phar:///composer.phar/vendor/react/promise/src/Promise.php:231
 React\Promise\Promise::React\Promise\{closure}() at phar:///composer.phar/vendor/react/promise/src/FulfilledPromise.php:42
 React\Promise\FulfilledPromise->done() at phar:///composer.phar/vendor/react/promise/src/Promise.php:66
 React\Promise\Promise::React\Promise\{closure}() at phar:///composer.phar/vendor/react/promise/src/Promise.php:168
 React\Promise\Promise->settle() at phar:///composer.phar/vendor/react/promise/src/Promise.php:231
 React\Promise\Promise::React\Promise\{closure}() at phar:///composer.phar/vendor/react/promise/src/FulfilledPromise.php:42
 React\Promise\FulfilledPromise->done() at phar:///composer.phar/vendor/react/promise/src/Promise.php:135
 React\Promise\Promise::React\Promise\{closure}() at phar:///composer.phar/vendor/react/promise/src/Promise.php:168
 React\Promise\Promise->settle() at phar:///composer.phar/vendor/react/promise/src/Promise.php:231
 React\Promise\Promise::React\Promise\{closure}() at phar:///composer.phar/vendor/react/promise/src/FulfilledPromise.php:42
 React\Promise\FulfilledPromise->done() at phar:///composer.phar/vendor/react/promise/src/Promise.php:135
 React\Promise\Promise::React\Promise\{closure}() at phar:///composer.phar/vendor/react/promise/src/Promise.php:168
 React\Promise\Promise->settle() at phar:///composer.phar/vendor/react/promise/src/Promise.php:231
 React\Promise\Promise::React\Promise\{closure}() at phar:///composer.phar/vendor/react/promise/src/FulfilledPromise.php:42
 React\Promise\FulfilledPromise->done() at phar:///composer.phar/vendor/react/promise/src/Promise.php:135
 React\Promise\Promise::React\Promise\{closure}() at phar:///composer.phar/vendor/react/promise/src/Promise.php:168
 React\Promise\Promise->settle() at phar:///composer.phar/vendor/react/promise/src/Promise.php:231
 React\Promise\Promise::React\Promise\{closure}() at n/a:n/a
 call_user_func() at phar:///composer.phar/src/Composer/Util/ProcessExecutor.php:321
 Composer\Util\ProcessExecutor->countActiveJobs() at phar:///composer.phar/src/Composer/Util/Loop.php:98
 Composer\Util\Loop->wait() at phar:///composer.phar/src/Composer/Installer/InstallationManager.php:496
 Composer\Installer\InstallationManager->waitOnPromises() at phar:///composer.phar/src/Composer/Installer/InstallationManager.php:469
 Composer\Installer\InstallationManager->executeBatch() at phar:///composer.phar/src/Composer/Installer/InstallationManager.php:390
 Composer\Installer\InstallationManager->downloadAndExecuteBatch() at phar:///composer.phar/src/Composer/Installer/InstallationManager.php:282
 Composer\Installer\InstallationManager->execute() at phar:///composer.phar/src/Composer/Installer.php:759
 Composer\Installer->doInstall() at phar:///composer.phar/src/Composer/Installer.php:281
 Composer\Installer->run() at phar:///composer.phar/src/Composer/Command/InstallCommand.php:137
 Composer\Command\InstallCommand->execute() at phar:///composer.phar/vendor/symfony/console/Command/Command.php:298
 Symfony\Component\Console\Command\Command->run() at phar:///composer.phar/vendor/symfony/console/Application.php:1015
 Symfony\Component\Console\Application->doRunCommand() at phar:///composer.phar/vendor/symfony/console/Application.php:299
 Symfony\Component\Console\Application->doRun() at phar:///composer.phar/src/Composer/Console/Application.php:334
 Composer\Console\Application->doRun() at phar:///composer.phar/vendor/symfony/console/Application.php:171
 Symfony\Component\Console\Application->run() at phar:///composer.phar/src/Composer/Console/Application.php:130
 Composer\Console\Application->run() at phar:///composer.phar/bin/composer:83
 require() at /composer.phar:29
install [--prefer-source] [--prefer-dist] [--prefer-install PREFER-INSTALL] [--dry-run] [--dev] [--no-suggest] [--no-dev] [--no-autoloader] [--no-progress] [--no-install] [-v|vv|vvv|--verbose] [-o|--optimize-autoloader] [-a|--classmap-authoritative] [--apcu-autoloader] [--apcu-autoloader-prefix APCU-AUTOLOADER-PREFIX] [--ignore-platform-req IGNORE-PLATFORM-REQ] [--ignore-platform-reqs] [--] [<packages>...]
```

**Comment by bytehead** [comment]
My guess: Composer 2.3.0 internally ships with a newer release of `symfony/process` which is not compatible anymore.

**Comment by bytehead** [comment]
The argument was already an `array` in `symfony/process: 4.4`, but not typehinted: https://github.com/symfony/process/blob/4.4/Process.php#L132

**Comment by bytehead** [comment]
Closed in favor of #88
